types/convert_test.go

```diff
@@ -243,6 +243,12 @@ func (s *testTypeConvertSuite) TestConvertType(c *C) {
 	v, err = Convert("-10000", ft)
 	c.Assert(terror.ErrorEqual(err, ErrOverflow), IsTrue, Commentf("err %v", err))
 	c.Assert(v.(*MyDecimal).String(), Equals, "-9999.9999")
+	v, err = Convert("1,999.00", ft)
+	c.Assert(terror.ErrorEqual(err, ErrBadNumber), IsTrue, Commentf("err %v", err))
+	c.Assert(v.(*MyDecimal).String(), Equals, "0.0000")
+	v, err = Convert("1,999,999.00", ft)
+	c.Assert(terror.ErrorEqual(err, ErrBadNumber), IsTrue, Commentf("err %v", err))
+	c.Assert(v.(*MyDecimal).String(), Equals, "0.0000")
```

Comment:
how about add a test case to cover 

```
set sql_mode='';
insert into t12 values('1,999.00');
```

result~?

Reply:
ok, can I add this test case in `/executor/write_test.go`?

Reply:
> how about add a test case to cover
> 
> ```
> set sql_mode='';
> insert into t12 values('1,999.00');
> ```
> 
> result~?

ok, can I add this test case in `/executor/write_test.go`?

Reply:
emm... yes~ maybe it's incompatible with mysql when in sql_mode=''..

Reply:
maybe we can ignore `sql_mode` first.. :D